specification/columns/chargesubcategory.md

```diff
@@ -57,7 +58,15 @@ Allowed values when ChargeCategory is "Adjustment":
 | Refund             | Negative charges that were previously billed and are being returned by the provider. Providers can have multiple types of refunds such as resolving a tax error or for returned or exchanged commitment-based discounts. |
 | Credit             | Negative charges granted by the provider for various scenarios, like negotiated benefits, usage discounts, or promotional credits.                                                                                       |
 | Rounding Error     | Positive or negative charges that are needed to ensure raw billing data aggregations match the total cost on the invoice, which may be rounded.                                                                   |
-| General Adjustment | Positive or negative charges the provider applies that do not fall into other adjustment category values.                                                                                                                |
+| Other              | Positive or negative charges the provider applies that do not fall into other adjustment category values.                                                                                                                |
+
+Allowed valuew when ChargeCategory is "Purchase"
+
+| Value              | Description     |
+| :------------------| :----------------------------------------------------------------------------------------|
+| Commitment         | Purchase relates to a commitment based purchase such as reservation, savings plan or partial upfront reservation or savings plan |
+| MarketPlace        | Purchase relates to a service purchased from the CSP marketplace |
```

Comment:
There are commitments for Marketplace offers, so this might be better off defined in a `PublisherCategory` category.